src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Config/FlagsConfig.cs

```diff
@@ -38,5 +38,37 @@ public IEnumerable<IOperationProvider> Process(IComponentManager componentManage
                 yield return new SetFlag(flag, on, off, onNoEmit, offNoEmit, id, @default);
             }
         }
+
+        public static readonly string FlagConditionalSuffix = ":cnd";
```

Comment:
No, it's not needed. I thought I'd need them exposed for something else, but don't.